src/d3a/models/area/market_rotators.py

```diff
@@ -15,16 +15,24 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
+from enum import Enum
 from logging import getLogger
 from typing import Dict
 
-from d3a import constants
 from d3a_interface.constants_limits import ConstSettings, GlobalConfig
 from pendulum import DateTime
 
+from d3a import constants
+
 log = getLogger(__name__)
 
 
+class MarketClassType(Enum):
+    SPOT = 0
+    BALANCING = 1
+    SETTLEMENT = 2
```

Comment:
Might better belongs to `d3a/src/d3a/models/market/__init__.py`

Reply:
Will move it thanks